jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -222,6 +222,17 @@ def on_limits_change(self, *args):
             return
         self.set_compass(self.state.layers[i].layer)
 
+    @property
+    def top_visible_data_label(self):
+        """Data label of the top visible layer in the viewer."""
+        try:
+            i = get_top_layer_index(self)
+        except IndexError:
+            data_label = ''
```

Comment:
probably doesn't need coverage?

Reply:
Doesn't hurt. I added a commit to cover this line, hopefully. Thanks for catching it!